webui/react/src/pages/WorkspaceDetails/WorkspaceMembers.tsx

```diff
@@ -11,31 +9,19 @@ import InteractiveTable, {
 import { getFullPaginationConfig } from 'components/Table/Table';
 import TableFilterSearch from 'components/Table/TableFilterSearch';
 import Avatar from 'components/UserAvatar';
-import { useStore } from 'contexts/Store';
 import useFeature from 'hooks/useFeature';
 import { useFetchKnownRoles } from 'hooks/useFetch';
```

Comment:
I think you can now remove this `useFetchKnownRoles` stuff since the roles are coming out of your new calls

Reply:
Removing this now.